aztec-nargo/Dockerfile

```diff
@@ -0,0 +1,19 @@
+# to get built nargo binary
+FROM aztecprotocol/noir as built-noir
+
+# to get built avm-transpiler binary
+FROM aztecprotocol/avm-transpiler as built-transpiler
+
+
+FROM ubuntu:noble
+# Install Tini as nargo doesn't handle signals properly.
+# Install git as nargo needs it to clone.
+RUN apt-get update && apt-get install -y git tini && rm -rf /var/lib/apt/lists/* && apt-get clean
```

Comment:
Copied from Noir's earthfile/dockerfile.... I don't know whether we need this or not.